pygmt/clib/session.py

```diff
@@ -1751,15 +1751,15 @@ def virtualfile_to_dataset(
 
         Parameters
         ----------
+        vfname
+            The virtual file name that stores the result data. Required for ``"pandas"``
+            and ``"numpy"`` output type.
         output_type
             Desired output type of the result data.
 
             - ``"pandas"`` will return a :class:`pandas.DataFrame` object.
             - ``"numpy"`` will return a :class:`numpy.ndarray` object.
             - ``"file"`` means the result was saved to a file and will return ``None``.
-        vfname
-            The virtual file name that stores the result data. Required for ``"pandas"``
-            and ``"numpy"`` output type.
         column_names
             The column names for the :class:`pandas.DataFrame` output.
```

Comment:
If the proof of concept at #3117 works, these lines will need to be updated (after we add some logic to use the header column names in this `virtualfile_to_dataset` method)